CONTRIBUTING.adoc

```diff
@@ -337,6 +337,10 @@ openshift.local.config) can be supplied instead:
 
         $ OPENSHIFT_CONFIG_ROOT=[cluster config root] test/extended/networking.sh
 
+It's also possible to supply the path to a kubeconfig file:
+
+        $ TEST_KUBECONFIG=/path/to/admin.kubeconfig test/extended/networking.sh
```

Comment:
Shouldn't this be `OPENSHIFT_KUBECONFIG` or `OPENSHIFT_TEST_KUBECONFIG` or something?